.github/workflows/beta-release.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish beta release when pushing to the main branch
+
+on:
+  push:
+    branches: [main]
```

Comment:
For now non blocking but this job needs at some point an exclusion for pushes or merges to main containing a commit or tag matching this pattern "v*". 

Otherwise we publish a release twice, once as full release and once as alpha/beta 

The other open question would be how we communicate fixes in those intermediate releases if we don't tag them, as most changelog generators work based on recent tags. Let's discuss this next week :)

Reply:
as discussed, lets merge this now and find a solution for the tagging and changelog-generation later